pxr/usd/plugin/usdAbc/alembicData.h

```diff
@@ -43,7 +44,7 @@ class UsdAbc_AlembicData : public SdfAbstractData {
     /// Returns a new \c UsdAbc_AlembicData object.  Outside a successful
     /// \c Open() and \c Close() pairing, the data acts as if it contains
     /// a pseudo-root prim spec at the absolute root path.
-    static UsdAbc_AlembicDataRefPtr New();
+    static UsdAbc_AlembicDataRefPtr New(const SdfFileFormat::FileFormatArguments* = nullptr);
```

Comment:
It introduces a couple of layers of extra complexity to change the higher-level pattern of passing FileFormatArguments by const-ref to by pointer, and I'm curious why we'd want to do it?  You're already paying for heap-allocating a copy of the std::map (which is generally going to be empty or very small), so why not eliminate all of that and the checks-for-nullptr, and just plumb the const-ref all the way down and let the member-var hold a copy?

Reply:
No idea why I chose that initially other than a quick way to enable disable the custom behavior.
Easy enough to change.